api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -125,21 +125,21 @@ public static String getOpenmrsProperty(String property) {
 		return null;
 	}
 		
-	public static String DATABASE_NAME = "openmrs";
+	public static final String DATABASE_NAME = "openmrs";
 	
-	public static String DATABASE_BUSINESS_NAME = "openmrs";
+	public static final String DATABASE_BUSINESS_NAME = "openmrs";
 	
```

Comment:
The above 2 are also reassigned later

Reply:
@wluyima Sorry, I couldn't find a place where DATABASE_BUSINESS_NAME is modified

Reply:
@dilekamadushan you should also run `mvn clean package` before you push your changes for review. You should get compile errors if you make something a constant which turns out to be reassigned.